extensions/amp-subscriptions/0.1/amp-subscriptions.js

```diff
@@ -505,7 +519,7 @@ export class SubscriptionService {
 
   /**
    * Evaluates platforms and select the one to be selected for login.
-   * @return {!./subscription-platform.SubscriptionPlatform}
+   * @return {!Promise<!./subscription-platform.SubscriptionPlatform>}
```

Comment:
I think this was intentionally synchronous to avoid any chance of a popup blocker.

Reply:
Ah, that's annoying, lets discuss the best way around this F2F

Reply:
I think it's moot because in every places it's called there is already a promise chain.  for example in `selectPlatformForLogin()` it ends up at [`delegateActionToService()`](https://github.com/ampproject/amphtml/blob/29ae999ecaed043251578265a82bef9cc95a0270/extensions/amp-subscriptions/0.1/amp-subscriptions.js#L477)